lang/src/loader_account.rs

```diff
@@ -206,3 +155,23 @@ impl<'info, T: ZeroCopy + Owner> Key for AccountLoader<'info, T> {
         *self.acc_info.key
     }
 }
+
+impl<'info, T> Deref for AccountLoader<'info, T>
+where
+    T: ZeroCopy + Owner,
+{
+    type Target = T;
+
+    fn deref(&self) -> &Self::Target {
+        unsafe { &(*self.data) }
```

Comment:
Can we add a comment explaining why these uses of `unsafe` are indeed safe.

Reply:
Ive added one, does it make sense?